src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -491,11 +504,14 @@ def estimateTime(self, remaining=None):
 
         acq_time = 0
         for stream in self._streams:
-            acq_stream_time = acqmng.estimateTime([stream])
+            # add 1s to account for stage/objective movement time in z direction
+            acq_stream_time = acqmng.estimateTime([stream]) + 1
```

Comment:
@K4rishma I think this line is wrong. I think it should be as below. What do you think?

```
        acq_time = 0
        for stream in self._streams:
            acq_stream_time = acqmng.estimateTime([stream])
            if stream.focuser is not None and len(self._zlevels) > 1:
                # Acquisition time for each stream will be multiplied by the number of zstack levels
                zlevels = [item for item in self._zlevels if item is not None]
                # add 1s to account for stage/objective movement time in z direction
                acq_stream_time = (acq_stream_time + 1) * len(zlevels)
            # add 2 seconds to account for switching from one tile to next tile
            acq_time += acq_stream_time + 2
```